### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by mansa-susa**
- Add an event listener to the mobile menu icon to display the menu mobile
- Add a close button to the mobile menu
- Add an event listener to the close button to close the mobile menu when clicked
- Add an anchor to the sections of the page that links to the menu links
- Add an event listener to the menu links to close the mobile menu pop-up when clicked

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.). Please inculde eslint to linters configuration.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
-menuBtn.addEventListener('click', displayMenu);
+// displayMenu function declaration //
+function displayMenu() {
+  menuBox.classList.toggle('menuDialog');
+}
 
-closeBtn.addEventListener('click', closeMenu);
+// closeMenu function declaration //
+function closeMenu() {
```
Comment (Ismailco) [inline main.js:14]: - [ ] Kindly fix the horizontal overflow on the page, so the close button on the mobile menu so the close button can be seen without scrolling left and your design look like the Figma design

|Your design|Figma design|
|-|-|
|Screenshot_20220914_172004|Screenshot_20220914_172212|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi Team, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — APPROVED** [review]
Hi @mansa-susa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏
👏👏👏


## Highlights
- [x] Great use of github flow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**